.github/workflows/cd.yaml

```diff
@@ -4,9 +4,28 @@ on:
   push:
     branches:
       - master
+      - eks-test-2
```

Comment:
This should be removed no?

Reply:
Yes, I should have taken it out. However, it does no harm, so not urgent.